kornia/geometry/transform/affwarp.py

```diff
@@ -670,7 +670,7 @@ class Resize(nn.Module):
 
     .. raw:: html
 
-        <gradio-app space="kornia/kornia-resize-antialias"></gradio-app>
+        <gradio-app src="kornia/kornia-resize-antialias"></gradio-app>
```

Comment:
just as a note, we can figure out in a further PR: on kornia HF we have 17 spaces, and I think not all of them are in the docs